app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @first_user = User.first
```

Comment:
- [ ] Here we don't have save it in any variable, we can directly return `User.first`.
```
def current_user
  User.first
end
```